### ISSUE TITLE
Please test out the openshift-ansible role we've prepared

### ISSUE THREAD

**Issue by tbielawa** [ManageIQ/manageiq-pods#146]
In https://github.com/openshift/openshift-ansible/pull/4041 I've prepared a basic working POC for a role that installs CFME as part of a new OCP installation.

The PR works on infra that meets the requirements outlined in this project's README. Installations on hardware below those requirements will fail. I did my tests in the OS1 OpenStack environment at work.

Please give it a test so we can get some preliminary feedback on this before we go into full on '1.0' and *polishing* mode.

Thanks in advance! :smiley:

**Comment by carbonin** [comment]
This looks great and thanks for the heads up @tbielawa (I didn't know this type of work was going on), but fair warning this repo is likely to undergo quite a few significant changes in the near future.

For example we removed the need for privileged pods in #140, but may need to add it back in some capacity to support parts of our external authentication (sssd) - the current thought is to separate another pod for the front-end of the app which will have to run privileged.

Additionally the entirety of the manageiq project is undergoing a re-architecture which will heavily rely on openshift. This will likely render this repo in its current form fairly unrecognizable (we are looking to move to a proper micro-services arch where each manageiq worker would probably be a separate image).

I just mention this because after a glance at https://github.com/openshift/openshift-ansible/pull/4041 I noticed you included an entire copy of our template. That will likely be a maintenance headache in the future.

**Comment by bazulay** [comment]
@carbonin  @fbladilo  The work @tbielawa was doing is for  deployment of podified manageiq 5.8 on origin 1.6,
It will need to be adopted once the arch changes towards 5.9, but this needs to be verified as well.

**Comment by fbladilo** [comment]
@tbielawa Thanks, this is cool stuff. What would be the quickest way to get testing this new ansible role? We currently work mainly on GA releases (1.5/3.5) in our clusters.

**Comment by tbielawa** [comment]
@fbladilo you need a cluster with 1 master and 2 nodes that meet the requirements outlined in the [README.md](https://github.com/ManageIQ/manageiq-pods/blob/master/README.md#cluster-sizing). I used the Red Hat OS1 cluster because it's cheaper ($free) than AWS for the same (huge) sized instances.

You can get started [here](https://github.com/tbielawa/openshift-ansible/blob/1c054134a4e12c98c6a9c425c20464509b91a0ec/roles/openshift_cfme/README.md#important-notes) in MY PR readme. It outlines requirements, explains assumptions, and at the end, shows you how to run it. I used a [playbook like this](https://gist.github.com/tbielawa/f8264f2ec485cb436daf667192f0c735) for testing.

But depending on your preferred deployment style, you might need to change the references to `brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888` to `registry.ops.openshift.com`

I.e.:

```
$ sed -i 's/brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/registry.ops.openshift.com/g' your-inventory-file
```

And then fix up the private/public hostnames and IPs, maybe `ansible_ssh_user` if it's not `cloud-user`, `rhsub_*` variables can go away if your hosts are already subscribed to a package provider and you don't need to use the `playbooks/byo/rhel_subscribe.yml` playbook.

**Comment by fbladilo** [comment]
@bazulay @carbonin Successfully tested the openshift-ansible MIQ role with @tbielawa help, works perfectly some minor commits pending to fix the tags to latest-fine (CFME 4.5) on template.
```
[root@franco-ocp-master openshift-ansible (cfme)]# oc get pods
NAME                 READY     STATUS    RESTARTS   AGE
manageiq-0           1/1       Running   0          16m
memcached-1-8ppjs    1/1       Running   0          15m
postgresql-1-bvc8k   1/1       Running   0          15m
[root@franco-ocp-master openshift-ansible (cfme)]# 
```
Testing was done on OCP 3.5 with a git clone of the new openshift-ansible miq role, the most significant paremeter on inventory file is : 

```
# CFME - enable during install
openshift_cfme_install_app=True
```

**Comment by tbielawa** [comment]
The role has been merged and it's been tested by your team members. Thanks for the help!
